src/pages/about.astro

```diff
@@ -193,10 +193,9 @@ const members = filterInactiveMembers(await getMembers());
         <h2>Can individuals join?</h2>
 
         <p
-          >While we've had companies-of-one join, we don't yet have a route to individual membership. We're discussing
-          whether to add this in
-          <a href="https://github.com/opensourcepledge/opensourcepledge.com/issues/21">issue #21</a>. Feel free to
-          join the conversation.</p
+          >While we do want to add the option for individual membership in the long term, doing this properly will
+          require a bit more work. For now, we encourage generous individuals to join as a company-of-one, by using a
+          company name and logo and following the usual steps.</p
```

Comment:
How about this?

```suggestion
          >No, we don't offer an individual membership category, because we are focused on unlocking funding from _companies_ to maintainers. That said, single-person companies are <a href="/join/"welcome to join</a>, and we also appreciate <a href="/endorsements/">endorsements</a> from individuals.</p
```